CSharp-Playwright-BrowserStack/CSharp-Playwright-BrowserStack.csproj

```diff
@@ -17,8 +17,8 @@
 			<PrivateAssets>all</PrivateAssets>
 		</PackageReference>
 		<PackageReference Include="NUnit3TestAdapter" Version="4.5.0" />
-		<PackageReference Include="Microsoft.Playwright" Version="1.36.0" />
-		<PackageReference Include="Microsoft.Playwright.NUnit" Version="1.36.0" />
+		<PackageReference Include="Microsoft.Playwright" Version="*" />
+		<PackageReference Include="Microsoft.Playwright.NUnit" Version="*" />
```

Comment:
We should keep it `1.*` right? Suppose Playwright releases v2 then the sample might break right?

Reply:
Keeping parity with other repos. Also a product requirement to use the latest playwright version. It's expected that the newer playwright version will break and we'll prioritise fixing backward compatibility for it.